From 564222f628febe09ec4d2fe5125d08498516594b Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Mon, 11 Jul 2016 15:21:34 +0300
Subject: [PATCH 38/38] ARM: i.MX6UL: add a missing cpu0 definition

Add a missing cpu0 definition with the correct
arm and soc regulators. It fixes the kernel boot hang
issue.

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 arch/arm/boot/dts/imx6ul-cl-som-imx6ul.dtsi | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/boot/dts/imx6ul-cl-som-imx6ul.dtsi b/arch/arm/boot/dts/imx6ul-cl-som-imx6ul.dtsi
index b2788a3..939da82 100644
--- a/arch/arm/boot/dts/imx6ul-cl-som-imx6ul.dtsi
+++ b/arch/arm/boot/dts/imx6ul-cl-som-imx6ul.dtsi
@@ -126,6 +126,11 @@
 	};
 };
 
+&cpu0 {
+	arm-supply = <&reg_arm>;
+	soc-supply = <&reg_soc>;
+};
+
 &clks {
 	assigned-clocks = <&clks IMX6UL_CLK_PLL4_AUDIO_DIV>;
 	assigned-clock-rates = <786432000>;
-- 
1.9.1

